lib/pod/command/why.rb

```diff
@@ -42,7 +42,7 @@ def validate!
       end
 
       def run
-        UI.puts 'Loading dependencies...'
+        warn 'Loading dependencies...'
```

Comment:
The `warn` is unfortunate because this isn't a warning. Maybe add a little helper called `print_status` or something and have that call `warn`?